### PR TITLE
Add syntax highlighting to code samples

### PR DESCRIPTION
**Pull request by c0bra**
Add syntax highlighting to HTML and JS code blocks in README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,6 +28,7 @@ A migration guide for newer releases of the plugin can be found [here](MIGRATION
 
 2. Modify the config.xml file to contain (replacing with your configuration settings):
 
+```html
```
Comment (rlepinski) [inline README.md]: xml?

**Comment by rlepinski** [comment]
Have you filled out a http://urbanairship.com/legal/contribution-agreement/ ?

File: README.md (hunk 1 above)
Comment (c0bra) [inline README.md]: Sorry yes, my mistake

**Comment by c0bra** [comment]
@rlepinski Nope, I haven't. If that's what it takes to do a couple small changes to a documentation page then I'll just close the PR. Not worth the effort; sorry!

**Pull request closed without merging**